docs/requirements.txt

```diff
@@ -6,3 +6,4 @@ sphinx==1.8.5
 sphinx_rtd_theme
 nbsphinx==0.8.7
 m2r
+jinja2<3.1
```

Comment:
why is the jinja pinning necessary here?

Reply:
Hopefully it's not, now that our doc stack has been updated.